ksp_plugin_adapter/burn_editor.cs

```diff
@@ -255,13 +255,32 @@ class BurnEditor : ScalingRenderer {
     specific_impulse_in_seconds_g0_ = range;
   }
 
-  private double final_time => initial_time_.value + duration_;
+  internal string FormatPreviousCoastDuration(double value) {
+    return FlightPlanner.FormatPositiveTimeSpan(TimeSpan.FromSeconds(value));
+  }
+
+  internal bool TryParsePreviousCoastDuration(string str, out double value) {
+    value = 0;
+    TimeSpan ts;
+    if (!FlightPlanner.TryParseTimeSpan(str, out ts)) {
+      return false;
+    }
+    value = ts.TotalSeconds;
+    return true;
+  }
+
+  private double previous_burn_final_time => previous_burn_?.final_time ??
+                                             plugin_.FlightPlanGetInitialTime(
+                                                 vessel_.id.ToString());
```

Comment:
This is a confusing name, since it may not be `previous_burn_.final_time`.
Call it `time_base`, this is what we call it in the UI anyway.

Reply:
Done.